include/cutlass/gemm/collective/xe_mma_w8a8.hpp

```diff
@@ -1,299 +1,236 @@
-/***************************************************************************************************

```

Comment:
Why is most of this file marked as changed? it disappears if I hide whitespace changes. Can this whitespace changes be reverted?

Reply:
Sorry, I had mentioned that in the PR description.

The file was using Windows line endings, BTW, and `git diff` locally looks a bit distracting for a file with Windows line-endings, so I had changed the file to have Unix line-endings.

But, as you advised, I'll revert the change in line-endings.